scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/GetCreatedActivitiesAction.java

```diff
@@ -30,11 +33,16 @@ public GetCreatedActivitiesAction(final Connection connection) throws SQLExcepti
 
   public Map<GoalId, List<ActivityDirectiveId>> get(final long analysisId) throws SQLException {
     this.statement.setLong(1, analysisId);
+    this.statement.setLong(2, analysisId);
     final var resultSet = this.statement.executeQuery();
 
     final var createdActivities = new HashMap<GoalId, List<ActivityDirectiveId>>();
     while (resultSet.next()) {
-      final var goalId = new GoalId(resultSet.getLong("goal_id"), resultSet.getLong("goal_revision"));
+      final var goalId = new GoalId(
+          resultSet.getLong("goal_id"),
+          resultSet.getLong("goal_revision"),
+          Optional.of(resultSet.getLong("goal_invocation_id"))
```

Comment:
The `Optional.of` wrapping should be removed

Reply:
We decided with @mattdailis and @skovati to leave this for clarity

Reply:
@skovati points out that GoalId has multiple responsibilities, so maybe the more appropriate refactor to do would be to split GoalId and GoalInvocationId

Reply:
Yeah I'd love to refactor this, I'll take a look on Monday

Reply:
I should specify that the reason I said to remove this `Optional.of` wrapping is because all other creations of `GoalId` do not include it, instead allowing the constructor to do the wrapping, making this invocation valid but inconsistent with the rest of the code.

Reply:
I only see two usages not explicitly wrapping with `Optional.of` and two usages letting the constructor wrap implicitly. I'll change the ones that explicitly wrap, if that's what you prefer.

Reply:
This will be resolved with https://github.com/NASA-AMMOS/aerie/issues/1551